src/requestBuilderUtils/GraphBaseClient.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+import { Client } from "../Client";
+import { GraphRequest } from "../GraphRequest";
+import { ClientOptions } from "../IClientOptions";
+import { GraphSDKConfig } from "./GraphSDKConfig";
+
+/**
+* @public
+* Base class for all Graph Service Client classes.
+* Extending the client.init and client.api methods.
+*/
+export class GraphBaseClient extends Client {
+    protected constructor(clientOptions: ClientOptions, graphSDK?: GraphSDKConfig) {
```

Comment:
this class might need to be in the service lib or you might get into an issue where you have a double inheritance in the service lib (needs to inherit from that and from the generated client)